src/components/date-range-picker.vue

```diff
@@ -10,7 +10,7 @@ including this file, may be copied, modified, propagated, or distributed
 except according to the terms contained in the LICENSE file.
 -->
 <template>
-  <label class="form-group">
+  <label class="form-group"><!-- eslint-disable-line vuejs-accessibility/label-has-for -->
```

Comment:
This `<label>` does contain an `<input>` via flatpickr, but ESLint doesn't know that.